core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -230,6 +230,9 @@ private[spark] class DAGScheduler(
     sc.getConf.getInt("spark.stage.maxConsecutiveAttempts",
       DAGScheduler.DEFAULT_MAX_CONSECUTIVE_STAGE_ATTEMPTS)
 
+  private[scheduler] val ignoreDecommissionFetchFailure =
```

Comment:
Could you add a comment like the following?
https://github.com/apache/spark/blob/2d6d5e2c44861745e4a7f5a4c15f74337f651b03/core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala#L226-L228